README.md

```diff
@@ -22,10 +22,20 @@ Ubuntu users should install these packages before building an image:
 apt-get install binfmt-support qemu-user-static make
 ```
 
+### Customization
+
+By exporting `OS_IMAGE` and `OS_URI` you can use a different operating system, like so:
+
+```bash
+export OS_IMAGE=root.tar.xz
+export OS_URI=http://vx2-downloads.raspberrypi.org/raspbian_lite/archive/${OS_IMAGE}
```

Comment:
Raspbian splits up boot and root into two xz files, not sure what th ebest way to deal with them both are